src/app/app.ts

```diff
@@ -1,9 +1,11 @@
+import infoContentProvider from './components/info'
 import { playAudio } from './utils'
 import { GamePage, HomePage } from './screens'
 
 export class App {
   start() {
     console.log('Starting application...')
+    new infoContentProvider()
```

Comment:
Why is it initialized here? I think it belongs to start page so it should be used there.